docs/rst/maintainer_guide.rst

```diff
@@ -302,6 +302,23 @@ need to update the lock file.
 
 .. _sec-maintainer_guide-release:
 
+Continuous integration
```

Comment:
Please move this section before the anchor, which is supposed to bring the reader to the release making instructions.